app/playlist/page.tsx

```diff
@@ -0,0 +1,91 @@
+"use client";
```

Comment:
We should try to keep the page component as server component. We can probably have the data fetching logic here and create child components components which can be client components